CMakeLists.txt

```diff
@@ -113,6 +113,10 @@ configure_file(templates/setup.sh.in ${CMAKE_CURRENT_BINARY_DIR}/setup.sh @ONLY)
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/setup.sh
     DESTINATION ${CMAKE_INSTALL_PREFIX}
```

Comment:
Any deprecation plans for this?

Reply:
In advance of 24.05, modify the template for setup.sh to warn if $0 is setup.sh instead of thisepic.sh.

Reply:
#700